scania-digital-design-system/sdds
# feat(banner): webcomponent

**Describe pull-request**  
Webcomponent for banner.

**Solving issue**  
Fixes: AB#3141

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below.
2. Check in Components -> Banner -> Web Component
3. Check that the styling is correct and the controls works as intended.

**Screenshots**  


**Additional context**
